### PR TITLE
Upgrading amo2kinto

### PR DESCRIPTION
**Pull request by Natim**
The hard part here is to keep some meaningful tests while not relying on json2kinto anymore.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ OBJECTS = .venv .coverage
 all: install
 install: $(INSTALL_STAMP)
 $(INSTALL_STAMP): $(PYTHON) setup.py
-	$(VENV)/bin/pip install -U pip
+	$(VENV)/bin/pip install -U pip kinto-wizard
```
Comment (leplatrem) [inline Makefile]: Should we add it to dev-requirements too?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -8,7 +8,15 @@ the version control of each dependency.
...
-- Nothing changed yet.
+amo2kinto
+'''''''''
+
+**amo2kinto 1.7.2 → 3.0.0**: https://github.com/mozilla-services/kinto-amo/releases/tag/3.0.0
+
+**Bug fixes**
+
+- Remove json2kinto importer
+- Remove xml verifier
```
Comment (leplatrem) [inline CHANGELOG.rst]: This will lead to 5.0 right?

File: Makefile (hunk 1 above)
Comment (Natim) [reply Makefile]: We don't have a dev-requirements.txt at the moment, reason why I added it here.

File: CHANGELOG.rst (hunk 2 above)
Comment (Natim) [reply CHANGELOG.rst]: Yes we could, also it doesn't change the kinto service behaviour so I don't think it is needed. amo2kinto is not a kinto plugin and the kinto-amo major didn't changed.

**Comment by glasserc** [comment]
This looks OK to me, but I'm not crazy about the 20kloc blob of YAML that we have to add. I don't really 100% understand why we have that. Is that meant to represent a test case for kinto-signer, with the signing workflow defined by a dump from a live AMO blocklist? If the AMO blocklist source gets updated, will we want to update it here too? If so, I think we should have some kind of fetching process rather than encode a single dump. If not, then I think we should isolate the part we actually want from the AMO blocklist and use that instead.

**Comment by Natim** [comment]
> Is that meant to represent a test case for kinto-signer

Yes it does an maybe we can actually shrink it down to 10 records per collections

> f the AMO blocklist source gets updated, will we want to update it here too

No we don't want to update it ever.

**Pull request merged**